src/document-converter.ts

```diff
@@ -159,30 +159,34 @@ function getImportDeclarations(
 export class DocumentConverter {
   private readonly originalUrl: OriginalDocumentUrl;
```

Comment:
private properties and methods by convention are prefixed with `_` and this is actually linted for in some of our other projects...  Maybe separate PR but `originalUrl` -> `_originalUrl` etc.

Reply:
Agreed, I'll tackle in a separate PR since there are a few different PRs in flight that this would affect.